backend-service/src/main/java/com/smartparkinglot/backendservice/controller/DriverController.java

```diff
@@ -34,10 +32,11 @@
         return new ResponseEntity<Driver>(driverService.addOrSave(driver),HttpStatus.CREATED);
     }
 
-    @PostMapping("/login") // form data olarak gelmeli
+    @GetMapping("/login") // form data olarak gelmeli
```

Comment:
Its good way to contains every request which has body, with `POST` method. My recommendation is
`@PostMapping`